gcd/build.gradle

```diff
@@ -1,16 +1,30 @@
-repositories {
-    mavenLocal()
-    mavenCentral()
+buildscript {
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+    }
+}
+
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
 }
 
 dependencies {
 
     // TODO:2015-11-19:alexander.litus: change Spine local dependencies to Maven central repository dependencies
```

Comment:
We still need examples dependency from mavenLocal. We can remove this comment if we deploy examples-0.3.